lib/index.js

```diff
@@ -32,30 +32,32 @@ Floodlight.ensure(function(msg){
  *
  * https://segment.com/docs/spec/track/
  *
- * @param {Track} track
- * @param {Object} settings
+ * @param {Object} ret
  * @param {Function} fn
  * @api private
  */
 
-Floodlight.prototype.track = function(payload, done){
+Floodlight.prototype.track = function(ret, done){
   // If the payload is empty, return invalid
-  if (is.empty(payload)) return this.invalid('Only mapped events will be sent to DoubleClick Floodlight.');
-  // FL wants you to send the parameters as an endpoint delimited by semi-colons rather than get query params
-  var endpoint = [
-   'dc_rdid=' + payload.dc_rdid,
-   'src=' + payload.src,
-   'cat=' + payload.cat,
-   'type=' + payload.type,
-   'ord=' + payload.ord,
-   'tag_for_child_directed_treatment=' + payload.tag_for_child_directed_treatment,
-   'dc_lat=' + payload.dc_lat
-  ];
-
-  endpoint = endpoint.join(';');
+  if (!ret.endpoints.length) return;
```

Comment:
Hm yeah but we have docs for that. You can also consider that if we throw an error, any unmapped track events in the debugger will have an error notification which probably isn't too sexy to look at especially since only a few specific events (conversions) are being mapped so I'd prefer not throwing an error and just defer to success and our docs for them to figure out that they have to map it. We can also make it more clear in the docs that defining the mappings is a requisite step.